static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -1,210 +1,180 @@
-        US: 'https://membership.theguardian.com/us/supporter',
-        AU: 'https://membership.theguardian.com/au/supporter',
-        INT: 'https://membership.theguardian.com/supporter'
-    };
-
-    // change messageCode to force redisplay of the message to users who already closed it.
-    // messageCode is also consumed by .../test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js
-    var messageCode = 'engagement-banner-2016-11-10';
-
-    var messages = {
-        UK: {
-            campaign: 'Coffee_49',
-            messageText: 'For less than the price of a coffee a week, you could help secure the Guardian’s future. Support our journalism for just £49 per year.'
-        },
-        US: {
-            campaign: 'control',
-            messageText: 'If you use it, if you like it, then why not pay for it? It’s only fair.'
-        },
-        AU: {
-            campaign: 'mem_au_banner',
-            messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia Member for just $100 a year.'
-        },
-        INT: {
-            campaign: 'mem_int_banner',
-            messageText: 'The Guardian’s voice is needed now more than ever. Support our journalism for just $69/€49 per year.'
+        'bean',
+        'qwery',
+        'common/utils/config',
+        'common/utils/storage',
+        'common/utils/template',
+        'common/modules/ui/message',
+        'text!common/views/membership-message.html',
+        'common/modules/commercial/commercial-features',
+        'common/modules/commercial/user-features',
+        'common/utils/mediator',
+        'Promise',
+        'common/utils/fastdom-promise',
+        'common/modules/experiments/ab',
+        'common/utils/$',
+        'common/views/svgs'
+    ], function (bean,
+                 qwery,
+                 config,
+                 storage,
+                 template,
+                 Message,
+                 messageTemplate,
+                 commercialFeatures,
+                 userFeatures,
+                 mediator,
+                 Promise,
+                 fastdom,
+                 ab,
+                 $,
+                 svgs) {
+
+        var endpoints = {
+            UK: 'https://membership.theguardian.com/uk/supporter',
+            US: 'https://membership.theguardian.com/us/supporter',
+            AU: 'https://membership.theguardian.com/au/supporter',
+            INT: 'https://membership.theguardian.com/supporter'
+        };
+
+        // change messageCode to force redisplay of the message to users who already closed it.
+        // messageCode is also consumed by .../test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js
+        var messageCode = 'engagement-banner-2016-11-10';
+        var notInTest = 'notintest';
+
+        var messages = {
+            UK: {
+                campaign: 'Coffee_49',
```

Comment:
Ideally, this would probably just be `'mem_uk_banner'` and be varied by an active test that changes things, to make it easier to spot the differences in the data. I don't know if case-sensitivity might also be an issue for data lake queries. As this code is already out, we'll have to leave it until any tests have completed.

The same would apply for the US message (below) too.